app/controllers/facility_user_reservations_controller.rb

```diff
@@ -0,0 +1,52 @@
+class FacilityUserReservationsController < ApplicationController
+
+  admin_tab :all
+  before_action :init_current_facility
+  before_action :authenticate_user!
+  before_action :check_acting_as
+  before_action :load_user
+
+  load_and_authorize_resource class: Reservation
+
+  layout "two_column"
+
+  # GET /facilities/:facility_id/users/:user_id/reservations
+  def index
+    @order_details = user_order_details
+                     .where("orders.ordered_at IS NOT NULL")
```

Comment:
I believe we have a scope for this. `purchased` (which is an alias of `ordered`, but I think we concluded that that got confusing against the idea of sorting).